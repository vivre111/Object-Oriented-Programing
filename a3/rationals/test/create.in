print a create a -45/1 print a create a -45/90 print a create b 2/1 print b create c 3/-4 print a print c create d 5/3 print d print c create a 3/2 print a
