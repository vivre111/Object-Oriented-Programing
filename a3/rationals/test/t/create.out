0
-45
-1/2
2
-1/2
-3/4
5/3
-3/4
3/2
